pkg/api/controller_test.go

```diff
@@ -6054,48 +6054,9 @@ func TestPeriodicGC(t *testing.T) {
 
 func TestSearchRoutes(t *testing.T) {
```

Comment:
Can you explain what these UT changes are?

Reply:
We needed tests that included authorization on global search. Unauthorized calls to be more precise. I refactored the code to allow calls using basic authorization. I kept only one user and changed his permissions instead of adding a second user with a different password because this operation can't be done dynamically (it requires a restart of the server).